src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/HaivisionX4DecoderCommunicator.java

```diff
@@ -1973,19 +2070,17 @@ private StreamInfo performCreateStreamControl(StreamInfo streamInfo) {
 				if (logger.isDebugEnabled()) {
 					logger.debug(request);
 				}
-				System.out.println(buildDeviceFullPath(DecoderURL.BASE_URI + DecoderURL.STREAMS));
-
 				StreamDataWrapper streamDataWrapper = doPost(buildDeviceFullPath(DecoderURL.BASE_URI + DecoderURL.STREAMS), request, StreamDataWrapper.class);
 				if (streamDataWrapper == null) {
-					throw new ResourceNotReachableException(DecoderConstant.CREATE_CONTROL_ERR);
+					throw new ResourceNotReachableException(DecoderConstant.CREATE_STREAM_CONTROL_ERR);
 				} else {
 					isCreatedStreamControl = true;
 				}
 			} else {
-				throw new ResourceNotReachableException(DecoderConstant.CREATE_CONTROL_ERR);
+				throw new ResourceNotReachableException(DecoderConstant.CREATE_STREAM_CONTROL_ERR);
 			}
 		} catch (Exception e) {
-			throw new ResourceNotReachableException(DecoderConstant.CREATE_CONTROL_ERR + e.getMessage());
+			throw new ResourceNotReachableException(DecoderConstant.CREATE_STREAM_CONTROL_ERR + e.getMessage());
```

Comment:
please check everywhere this issue, e should be the second arg

Reply:
I fixed it